lib/widget/form_textfield.dart

```diff
@@ -157,8 +157,7 @@ class TextInputController extends FormFieldController {
   {
```

Comment:
why even have this function? like all the setters you can do 
'maxLines' : (value) => _controller.maxLines = Utils.getInt(value, min: 1, fallback: 1)
Then remove the default value of 1 when you declare maxLines.

Please test your changes though.

Reply:
its not working its always null when maxLines is empty or null

Reply:
the line I have has a fallback, which forces maxLines to be 1 if the user doesn't enter anything. That will work yes?